src/test/java/com/marklogic/mgmt/TestConfig.java

```diff
@@ -17,13 +17,31 @@
 public class TestConfig {
 
     @Value("${mlManageHost:localhost}")
-    private String mlManageHost;
+    private String host;
```

Comment:
This class is used to read in properties from test.properties to control which ML instance the tests run against. I added these new properties so that I can do manual tests against an ML Cloud instance.